sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -275,15 +275,8 @@ class ProgramManager {
   /// Maps names of kernels to their unique kernel IDs.
   /// TODO: Use std::unordered_set with transparent hash and equality functions
   ///       when C++20 is enabled for the runtime library.
-  /// Access must be guarded by the m_KernelIDsMutex mutex
   std::unordered_map<std::string, kernel_id> m_KernelIDs;
```

Comment:
A note should be placed here saying that write access is only allowed during start-up process in `addImages`.

Reply:
Good idea. I've added a note.